vispy/visuals/markers.py

```diff
@@ -615,6 +585,35 @@ class MarkersVisual(Visual):
     vbar, hbar, cross, tailed_arrow, x, triangle_up, triangle_down,
     and star.
     """
+    _marker_funcs = {
```

Comment:
`_marker_name_to_func` maybe?

Reply:
I don't think that's needed, otherwise we should also do `_raycasting_method_to_function` and so on in `Volume`, and that seems redundant to me :) I'd rather change this to `_marker_shapes`.

Reply:
you're right! either is good, think I prefer marker funcs :)